mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -448,7 +448,8 @@ def Vector_ShuffleOp
     * the mask length equals the leading dimension size of the result
     * numbering the input vector indices left to right across the operands, all
       mask values must be within range, viz. given two k-D operands v1 and v2
-      above, all mask values are in the range [0,s_1+t_1)
+      above, all mask values are in the range [0,s_1+t_1). -1 is used to
+      represent a poison mask value.
```

Comment:
Can you add a sentence of explanation of how a poison mask value translates to the corresponding vector elements? The llvm langref does a good job with this IMO: https://llvm.org/docs/LangRef.html#id192